cmd/pbworker/main.go

```diff
@@ -53,7 +51,6 @@ func main() {
 	logger = logger.With().
 		Timestamp().
 		Str("hostname", hostname).
-		Str("worker_id", xid.New().String()).
```

Comment:
Oh it was because it is practically useless - in Kibana there are several fields which distinguish pods:

* pod
* hostname
* instance
* others I think too